File: src/G1_py/PYA108.py
from math import pow,sqrt

x1=eval(input())
y1=eval(input())
x2=eval(input())
y2=eval(input())

x_pow=pow(x1-x2,2)
y_pow=pow(y1-y2,2)
sum1=x_pow+y_pow
sqrt1=sqrt(sum1)

print("( {:} , {:} )".format(x1,y1))
print("( {:} , {:} )".format(x2,y2))
print("Distance = {:.4f}".format(sqrt1))

File: src/G1_rust/PYA108.rs
use std::io;

fn main(){
    let mut input: String = String::new();
    io::stdin().read_line(&mut input).unwrap();
    let x1: f64 = input.trim().parse().unwrap();
    input.clear();
    io::stdin().read_line(&mut input).unwrap();
    let x2: f64 = input.trim().parse().unwrap();
    input.clear();
    io::stdin().read_line(&mut input).unwrap();
    let y1: f64 = input.trim().parse().unwrap();
    input.clear();
    io::stdin().read_line(&mut input).unwrap();
    let y2: f64 = input.trim().parse().unwrap();
    
    let x_pow: f64 = (x1-x2).powi(2);
    let y_pow: f64 = (y1-y2).powi(2);
    let sum: f64 = x_pow + y_pow;
    let out: f64 = sum.sqrt();

    println!("( {} , {} )",x1,y1);
    println!("( {} , {} )",x2,y2);
    println!("Distance = {:.4}",out);
}